src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilder.java

```diff
@@ -94,6 +94,8 @@
  */
 public class SharedTableBuilder extends SharedTableCustomDynamicBuilder {
 
+    private static final String hashKeyField = "hk";
```

Comment:
presumably ctrl-alt-L doesn't pick it up, but does checkstyle notice when the variable name for a static final isn't written with underscore-separated all caps?

Reply:
Yeah, I didn't look it up, but from observation it appears that, with static finals, it doesn't require underscore-separated caps and also allows consecutive caps that wouldn't be allowed if in a non-static variable.